### ISSUE TITLE
Signal test fail with SSL Error 

### ISSUE THREAD

**Issue by arky** [ooni/probe#2459]
**Describe the bug**

Signal connectivity test are failing with following error. This might be geographically specific, signal test might work from other regions. 

```    78.57% signal: measure https://storage.signal.org/: ssl_unknown_authority 
   82.14% signal: measure https://cdn.signal.org/: ssl_unknown_authority 
   85.71% signal: measure https://cdn2.signal.org/: ssl_unknown_authority 
   89.29% signal: measure https://sfu.voip.signal.org/: ssl_unknown_authority 
   92.86% signal: measure https://textsecure-service.whispersystems.org/: ssl_unknown_authority 
   96.43% signal: measure dnslookup://uptime.signal.org: ok 
   100.00% signal: measure https://api.directory.signal.org/: ssl_unknown_authority 
```
**To Reproduce**

$ ./ooniprobe run im
   • Running im tests
[engine] iplookup: using ubuntu
[engine] sessionresolver: https://mozilla.cloudflare-dns.com/dns-query... ok
[engine] sessionresolver: http3://mozilla.cloudflare-dns.com/dns-query... ok
[engine] session: using probe services: {Address:https://api.ooni.io/ Type:https Front:}
   3.57% facebook_messenger: measure tcpconnect://external.xx.fbcdn.net:443: ok 
   7.14% facebook_messenger: measure tcpconnect://b-graph.facebook.com:443: ok 
   10.71% facebook_messenger: measure tcpconnect://scontent.xx.fbcdn.net:443: ok 
   14.29% facebook_messenger: measure tcpconnect://edge-mqtt.facebook.com:443: ok 
   17.86% facebook_messenger: measure tcpconnect://star.c10r.facebook.com:443: ok 
   21.43% facebook_messenger: measure tcpconnect://b-api.facebook.com:443: ok 
   25.00% facebook_messenger: measure dnslookup://stun.fbsbx.com: ok 
   26.79% telegram: measure http://149.154.167.51/: ok 
   28.57% telegram: measure http://149.154.175.100/: ok 
   30.36% telegram: measure http://149.154.175.50/: ok 
   32.14% telegram: measure http://149.154.171.5/: ok 
   33.93% telegram: measure http://149.154.167.91/: ok 
   35.71% telegram: measure http://95.161.76.100/: ok 
   37.50% telegram: measure http://149.154.175.50:443/: ok 
   39.29% telegram: measure http://149.154.167.51:443/: ok 
   41.07% telegram: measure http://149.154.171.5:443/: ok 
   42.86% telegram: measure http://149.154.175.100:443/: ok 
   44.64% telegram: measure http://149.154.167.91:443/: ok 
   46.43% telegram: measure http://95.161.76.100:443/: ok 
   48.21% telegram: measure https://web.telegram.org/: ok 
   50.00% telegram: measure http://web.telegram.org/: ok 
   50.71% whatsapp: measure tcpconnect://e3.whatsapp.net:443: ok 
   51.43% whatsapp: measure tcpconnect://e15.whatsapp.net:5222: ok 
   52.14% whatsapp: measure tcpconnect://e1.whatsapp.net:5222: ok 
   52.86% whatsapp: measure tcpconnect://e15.whatsapp.net:443: ok 
   53.57% whatsapp: measure tcpconnect://e12.whatsapp.net:5222: ok 
   54.29% whatsapp: measure tcpconnect://e2.whatsapp.net:443: ok 
   55.00% whatsapp: measure tcpconnect://e3.whatsapp.net:5222: ok 
   55.71% whatsapp: measure tcpconnect://e8.whatsapp.net:443: ok 
   56.43% whatsapp: measure tcpconnect://e16.whatsapp.net:5222: ok 
   57.14% whatsapp: measure tcpconnect://e9.whatsapp.net:443: ok 
   57.86% whatsapp: measure tcpconnect://e1.whatsapp.net:443: ok 
   58.57% whatsapp: measure tcpconnect://e12.whatsapp.net:443: ok 
   59.29% whatsapp: measure tcpconnect://e4.whatsapp.net:5222: ok 
   60.00% whatsapp: measure tcpconnect://e7.whatsapp.net:5222: ok 
   60.71% whatsapp: measure tcpconnect://e9.whatsapp.net:5222: ok 
   61.43% whatsapp: measure tcpconnect://e5.whatsapp.net:5222: ok 
   62.14% whatsapp: measure tcpconnect://e5.whatsapp.net:443: ok 
   62.86% whatsapp: measure tcpconnect://e13.whatsapp.net:443: ok 
   63.57% whatsapp: measure tcpconnect://e8.whatsapp.net:5222: ok 
   64.29% whatsapp: measure http://web.whatsapp.com/: ok 
   65.00% whatsapp: measure tcpconnect://e6.whatsapp.net:5222: ok 
   65.71% whatsapp: measure https://web.whatsapp.com/: ok 
   66.43% whatsapp: measure tcpconnect://e4.whatsapp.net:443: ok 
   67.14% whatsapp: measure tcpconnect://e7.whatsapp.net:443: ok 
   67.86% whatsapp: measure tcpconnect://e2.whatsapp.net:5222: ok 
   68.57% whatsapp: measure tcpconnect://e14.whatsapp.net:443: ok 
   69.29% whatsapp: measure tcpconnect://e10.whatsapp.net:5222: ok 
   70.00% whatsapp: measure tcpconnect://e11.whatsapp.net:5222: ok 
   70.71% whatsapp: measure tcpconnect://e6.whatsapp.net:443: ok 
   71.43% whatsapp: measure tcpconnect://e11.whatsapp.net:443: ok 
   72.14% whatsapp: measure tcpconnect://e10.whatsapp.net:443: ok 
   72.86% whatsapp: measure tcpconnect://e13.whatsapp.net:5222: ok 
   73.57% whatsapp: measure tcpconnect://e16.whatsapp.net:443: ok 
   74.29% whatsapp: measure tcpconnect://e14.whatsapp.net:5222: ok 
   75.00% whatsapp: measure https://v.whatsapp.net/v2/register: ok 
   78.57% signal: measure https://storage.signal.org/: ssl_unknown_authority 
   82.14% signal: measure https://cdn.signal.org/: ssl_unknown_authority 
   85.71% signal: measure https://cdn2.signal.org/: ssl_unknown_authority 
   89.29% signal: measure https://sfu.voip.signal.org/: ssl_unknown_authority 
   92.86% signal: measure https://textsecure-service.whispersystems.org/: ssl_unknown_authority 
   96.43% signal: measure dnslookup://uptime.signal.org: ok 
   100.00% signal: measure https://api.directory.signal.org/: ssl_unknown_authority 
[engine] sessionresolver: [{"URL":"https://dns.quad9.net/dns-query","Score":1},{"URL":"https://mozilla.cloudflare-dns.com/dns-query","Score":1},{"URL":"http3://mozilla.cloudflare-dns.com/dns-query","Score":0.9999999999572915},{"URL":"http3://cloudflare-dns.com/dns-query","Score":0.5213429036187203},{"URL":"http3://dns.google/dns-query","Score":0.3119594843271943},{"URL":"https://cloudflare-dns.com/dns-query","Score":0.2497022650757168},{"URL":"https://dns.google/dns-query","Score":0.1},{"URL":"system:///","Score":0}]
Steps to reproduce the behavior.

**Expected behavior**

**Screenshots**

**System information (please complete the following information):**

- Device: [e.g., iPhone6]
 - OS: [e.g., iOS8.1]
 - OONI Probe version: [e.g., 3.11.0]

**Additional context**

**Comment by arky** [comment]
Additional testing 

``` $ ./miniooni -i https://storage.signal.org/ web_connectivity@v0.5
[      0.000031] <info> Current time: 2023-04-21 15:05:39 +07
[      0.000076] <info> miniooni home directory: $HOME/.miniooni
[      0.000182] <info> Looking up OONI backends; please be patient...
[      0.172542] <info> sessionresolver: https://cloudflare-dns.com/dns-query... ok
[      0.751701] <info> session: using probe services: {Address:https://api.ooni.io/ Type:https Front:}
[      0.751751] <info> Looking up your location; please be patient...
[      0.751866] <info> iplookup: using cloudflare
[      0.819256] <info> sessionresolver: https://cloudflare-dns.com/dns-query... ok
[      1.127984] <info> - country: KH
[      1.128000] <info> - network: COGETEL Co., Ltd (AS23673)
[      1.128004] <info> - resolver's IP: 162.158.169.30
[      1.128009] <info> - resolver's network: Cloudflare, Inc. (AS13335)
[      1.128039] <info> [1/1] running with input: https://storage.signal.org/
[      1.161358] <info> [#3] lookup storage.signal.org using system... ok
[      1.192981] <info> [#2] lookup storage.signal.org using 8.8.4.4:53... ok
[      1.498771] <info> [#1] lookup storage.signal.org using https://dns.quad9.net/dns-query... ok
[      2.068193] <info> DNS whoami for 8.8.4.4:53/udp resolver: [{Address:172.217.44.201}]
[      2.098241] <info> DNS whoami for system resolver: [{Address:162.158.169.30}]
[      2.098317] <info> using resolved addrs: [{Addr:2404:6800:4005:820::2013 Flags:2} {Addr:172.217.25.211 Flags:4} {Addr:2404:6800:4001:80b::2013 Flags:4} {Addr:142.251.12.121 Flags:1} {Addr:2404:6800:4003:c0f::79 Flags:1} {Addr:142.250.207.83 Flags:2}]
[      2.098346] <info> prioritySelector: create with [{Addr:2404:6800:4005:820::2013 Flags:2} {Addr:172.217.25.211 Flags:4} {Addr:2404:6800:4001:80b::2013 Flags:4} {Addr:142.251.12.121 Flags:1} {Addr:2404:6800:4003:c0f::79 Flags:1} {Addr:142.250.207.83 Flags:2}]
[      2.098512] <info> [#4] GET https://storage.signal.org/ using [2404:6800:4005:820::2013]:443... network_unreachable
[      2.098576] <info> [#8] GET https://storage.signal.org/ using [2404:6800:4003:c0f::79]:443... network_unreachable
[      2.098585] <info> [#6] GET https://storage.signal.org/ using [2404:6800:4001:80b::2013]:443... network_unreachable
[      2.221660] <info> [#7] GET https://storage.signal.org/ using 142.251.12.121:443... ssl_unknown_authority
[      2.223219] <info> [#5] GET https://storage.signal.org/ using 172.217.25.211:443... ssl_unknown_authority
[      2.234363] <info> [#9] GET https://storage.signal.org/ using 142.250.207.83:443... ssl_unknown_authority
[      2.453977] <info> sessionresolver: https://cloudflare-dns.com/dns-query... ok
[      2.599272] <info> control for https://storage.signal.org/ using https://2.th.ooni.org/... in progress
[      3.703131] <info> control for https://storage.signal.org/ using https://2.th.ooni.org/... ok
[      3.703186] <info> additional addrs discovered by the TH: [142.250.180.19 2a00:1450:4009:81e::2013]
[      3.703513] <info> [#11] GET https://storage.signal.org/ using [2a00:1450:4009:81e::2013]:443... network_unreachable
[      4.204577] <info> [#10] GET https://storage.signal.org/ using 142.250.180.19:443... in progress
[      4.358386] <info> [#10] GET https://storage.signal.org/ using 142.250.180.19:443... ssl_unknown_authority
[      4.358611] <info> DNS: address 2404:6800:4005:820::2013: not resolved by TH
[      4.358647] <info> DNS: address 142.250.207.83: not resolved by TH
[      4.358669] <info> DNS: address 172.217.25.211: not resolved by TH
[      4.358689] <info> DNS: address 2404:6800:4001:80b::2013: not resolved by TH
[      4.358706] <info> DNS: address 142.251.12.121: not resolved by TH
[      4.358722] <info> DNS: address 2404:6800:4003:c0f::79: not resolved by TH
[      4.358762] <info> DNS: address 142.250.207.83: cannot confirm using TLS handshake
[      4.358776] <info> DNS: address 172.217.25.211: cannot confirm using TLS handshake
[      4.358792] <info> DNS: address 142.251.12.121: cannot confirm using TLS handshake
[      4.365664] <info> DNS: addr 142.250.207.83 has AS15169
[      4.365693] <info> DNS: addr 172.217.25.211 has AS15169
[      4.365712] <info> DNS: addr 142.251.12.121 has AS15169
[      4.365731] <info> DNS: addr 142.250.180.19 has AS15169
[      4.365750] <info> DNS: addr 2a00:1450:4009:81e::2013 has AS15169
[      4.365770] <info> DNSConsistency: consistent
[      4.365803] <info> website likely down: all TLS handshake attempts failed for both probe and TH
[      4.365808] <info> WEBSITE_DOWN_TLS: flags=0, accessible=false, blocking=false
[      4.367771] <info> submitting measurement to OONI collector; please be patient...
[      4.556042] <info> New reportID: 20230421T080544Z_webconnectivity_KH_23673_n1_1mekDGpVsbqXpio0
[      4.937615] <info> saving measurement to disk
[      4.945115] <info> experiment: recv   0.00  byte, sent   0.00  byte
[      4.945314] <info> sessionresolver: [{"URL":"https://cloudflare-dns.com/dns-query","Score":1},{"URL":"http3://mozilla.cloudflare-dns.com/dns-query","Score":0.9999900001},{"URL":"http3://cloudflare-dns.com/dns-query","Score":0.9999000010000001},{"URL":"https://dns.google/dns-query","Score":0},{"URL":"https://dns.quad9.net/dns-query","Score":0},{"URL":"http3://dns.google/dns-query","Score":0},{"URL":"https://mozilla.cloudflare-dns.com/dns-query","Score":0},{"URL":"system:///","Score":0}]
[      4.945368] <info> whole session: recv   8.20 kbyte, sent  34.07 kbyte ```

**Comment by arky** [comment]
Checking with helper: 

``` 
$ ./oohelper -server https://0.th.ooni.org -target https://cdn.signal.org
{
    "tcp_connect": {
        "52.222.139.17:443": {
            "status": true,
            "failure": null
        },
        "52.222.139.33:443": {
            "status": true,
            "failure": null
        },
        "52.222.139.42:443": {
            "status": true,
            "failure": null
        },
        "52.222.139.47:443": {
            "status": true,
            "failure": null
        },
        "65.9.17.107:443": {
            "status": true,
            "failure": null
        },
        "65.9.17.113:443": {
            "status": true,
            "failure": null
        },
        "65.9.17.30:443": {
            "status": true,
            "failure": null
        },
        "65.9.17.32:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:1200:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:2800:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:5600:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:8600:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:8800:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:ca00:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:e800:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:203f:ec00:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:1200:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:4a00:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:8000:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:a600:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:aa00:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:e000:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:ea00:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        },
        "[2600:9000:2204:ec00:1d:4f32:50c0:93a1]:443": {
            "status": true,
            "failure": null
        }
    },
    "tls_handshake": {
        "52.222.139.17:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "52.222.139.33:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "52.222.139.42:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "52.222.139.47:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "65.9.17.107:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "65.9.17.113:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "65.9.17.30:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "65.9.17.32:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:1200:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:2800:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:5600:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:8600:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:8800:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:ca00:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:e800:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:203f:ec00:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:1200:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:4a00:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:8000:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:a600:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:aa00:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:e000:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:ea00:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2600:9000:2204:ec00:1d:4f32:50c0:93a1]:443": {
            "server_name": "cdn.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        }
    },
    "quic_handshake": {},
    "http_request": {
        "body_length": -1,
        "discovered_h3_endpoint": "",
        "failure": "unknown_error",
        "title": "",
        "headers": {},
        "status_code": -1
    },
    "http3_request": null,
    "dns": {
        "failure": null,
        "addrs": [
            "52.222.139.42",
            "52.222.139.17",
            "52.222.139.33",
            "52.222.139.47",
            "2600:9000:2204:ea00:1d:4f32:50c0:93a1",
            "2600:9000:2204:e000:1d:4f32:50c0:93a1",
            "2600:9000:2204:8000:1d:4f32:50c0:93a1",
            "2600:9000:2204:ec00:1d:4f32:50c0:93a1",
            "2600:9000:2204:4a00:1d:4f32:50c0:93a1",
            "2600:9000:2204:aa00:1d:4f32:50c0:93a1",
            "2600:9000:2204:1200:1d:4f32:50c0:93a1",
            "2600:9000:2204:a600:1d:4f32:50c0:93a1"
        ]
    },
    "ip_info": {
        "2600:9000:203f:1200:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:2800:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:5600:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:8600:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:8800:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:ca00:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:e800:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:203f:ec00:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 1
        },
        "2600:9000:2204:1200:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:4a00:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:8000:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:a600:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:aa00:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:e000:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:ea00:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "2600:9000:2204:ec00:1d:4f32:50c0:93a1": {
            "asn": 16509,
            "flags": 2
        },
        "52.222.139.17": {
            "asn": 16509,
            "flags": 2
        },
        "52.222.139.33": {
            "asn": 16509,
            "flags": 2
        },
        "52.222.139.42": {
            "asn": 16509,
            "flags": 2
        },
        "52.222.139.47": {
            "asn": 16509,
            "flags": 2
        },
        "65.9.17.107": {
            "asn": 16509,
            "flags": 1
        },
        "65.9.17.113": {
            "asn": 16509,
            "flags": 1
        },
        "65.9.17.30": {
            "asn": 16509,
            "flags": 1
        },
        "65.9.17.32": {
            "asn": 16509,
            "flags": 1
        }
    }
}

$ ./oohelper -server https://0.th.ooni.org -target https://storage.signal.org
{
    "tcp_connect": {
        "142.250.207.83:443": {
            "status": true,
            "failure": null
        },
        "142.251.36.51:443": {
            "status": true,
            "failure": null
        },
        "[2404:6800:4005:820::2013]:443": {
            "status": true,
            "failure": null
        },
        "[2a00:1450:400e:810::2013]:443": {
            "status": true,
            "failure": null
        }
    },
    "tls_handshake": {
        "142.250.207.83:443": {
            "server_name": "storage.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "142.251.36.51:443": {
            "server_name": "storage.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2404:6800:4005:820::2013]:443": {
            "server_name": "storage.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        },
        "[2a00:1450:400e:810::2013]:443": {
            "server_name": "storage.signal.org",
            "status": false,
            "failure": "ssl_unknown_authority"
        }
    },
    "quic_handshake": {},
    "http_request": {
        "body_length": -1,
        "discovered_h3_endpoint": "",
        "failure": "unknown_error",
        "title": "",
        "headers": {},
        "status_code": -1
    },
    "http3_request": null,
    "dns": {
        "failure": null,
        "addrs": [
            "142.251.36.51",
            "2a00:1450:400e:810::2013"
        ]
    },
    "ip_info": {
        "142.250.207.83": {
            "asn": 15169,
            "flags": 1
        },
        "142.251.36.51": {
            "asn": 15169,
            "flags": 2
        },
        "2404:6800:4005:820::2013": {
            "asn": 15169,
            "flags": 1
        },
        "2a00:1450:400e:810::2013": {
            "asn": 15169,
            "flags": 2
        }
    }
}

```

**Comment by hellais** [comment]
This issue has been fixed since OONI Probe engine version 3.16.5 (see: https://github.com/ooni/probe/issues/2344). 

You appear to be running an out of date version of OONI Probe.

Please upgrade to the latest release.
